packages/api/package.json

```diff
@@ -30,7 +30,9 @@
     "web3": "^1.5.0"
   },
   "files": [
-    "/dist/**/*.js"
+    "build",
+    "src",
```

Comment:
I might my wrong, but isn't the `build` folder enough? or is `src` needed for types?

Reply:
ive removed all folders but the build folder